### DISCUSSION TITLE
Database empty

### DISCUSSION THREAD

**Discussion by Jan324663** [magicbug/Cloudlog#2017] [General]
Hello,

I have updated my old installation to version 2.0 as described here: https://github.com/magicbug/Cloudlog/wiki/Updating
I have chosen the ZIP variant.
Now my database seems to be empty.
My login is still there but my log is empty.

What can I do?
Jan

**Comment by phl0** [comment]
Have you setzp a station logbook and assigned a station location?

**Reply by Jan324663** [reply]
I had it before the update. Now it seems to be gone.
